tests/unit/workflowEngineDataSource/WorkflowEngineDataSourceQueuePopulator.spec.js

```diff
@@ -196,6 +188,90 @@ describe('workflow engine queue populator', () => {
             });
     });
 
+    it('refcounting shall work', done => {
+        const value = {
+            workflowId: '24b0af11-a366-4e64-8c7a-7c637ec1b93b',
+            workflowVersion: 42,
+            subType: 'basic',
+            bucket: 'foo',
+            key: 'bar',
+            nextNodes: []
+        };
+        const fdName = wedsqp._getHashString(
+            value.workflowId, value.workflowVersion);
+        const data = new Buffer(JSON.stringify(value));
+        assert(wedsqp._getFilterDescriptorsLength() === 0);
+        const zkc1 = new ZookeeperMock();
+        zkc1.connect();
+        const zkc2 = new ZookeeperMock();
+        zkc2.connect();
+        async.waterfall([
```

Comment:
You can use `async.series` since you don't have any results to forward from task to task.